locales/en.yml

```diff
@@ -75,7 +75,9 @@ en:
 
           Service:
               A service provided by sccli or systemd. For example:
-               docker, k8s, or openshift etc.
+               sccli service like docker, kubernetes, openshift etc
+               or
+               systemd service like libvirtd etc
```

Comment:
> This is imo not consistent with for example vagrant service-manager status

@hferentschik : I am not able to recollect why supporting systemd services are not consistent with `vagrant service-manager status`